### ISSUE TITLE
Make "Force Size" a permanent setting.

### ISSUE THREAD

**Issue by stroobandt** [connectbot/connectbot#1053]
## Is your feature request related to a problem? Please describe.
Without force sizing my terminal, terminal output will be rendered below my software keyboard (Hacker's keyboard) where it is not visible.

Each time I connect to a system, I have to manually set `Force Size` for the terminal. This requires a multitude of screen taps. Moreover, the `Force Size` UI has a bug which often requires to set the `Force Size` twice for it to take effect.

## Describe the solution you'd like
The `Force Size` setting should at least be a general ConnectBot setting that remains permanent.
After all, most users will not suddenly change the screen size of their device.

## Describe alternatives you've considered
In a further refinement, the `Force Size` setting could be made a permanent setting for each known connection separately, overriding the permanent general setting.

Another option would be to automatically detect the remaining screen size when the software keyboard is fully deployed. My software keyboard (Hacker's keyboard) requires an additional tap on a keyboard icon to fully deploy, which is also annoying.

In summary, too many manual taps are required each and every time before I can use the terminal after a connection is established.
